blobstorage/templates/_helpers.tpl

```diff
@@ -16,6 +16,10 @@ app.kubernetes.io/instance: {{ template "hmcts.blobstorage.releaseName" . }}
 {{- end -}}
 {{- end -}}
 
+{{- define "hmcts.blobstorage.storageAccountSha" -}}
+{{- include "hmcts.blobstorage.releaseName" . | sha256sum -}}
+{{- end -}}
+
 {{- define "hmcts.blobstorage.storageAccountName" -}}
```

Comment:
this should have it's version increased if you're making a change to make sure that there's no bundled charts colliding as this is global scoped

```suggestion
{{- define "hmcts.blobstorage.storageAccountName.v1" -}}
```